docs/src/development/local/lando.md

```diff
@@ -1,46 +1,124 @@
+For a complete reference, consult the following resources:
 
-If you do not yet have your project checked out, run:
+- [Lando Platform.sh plugin documentation](https://docs.lando.dev/config/platformsh.html) and [source code](https://github.com/lando/platformsh)
+- [Lando documentation](https://docs.lando.dev/)
 
-```bash
-$ lando init --source platformsh
-```
+## Before you begin
 
-That will offer an interactive dialog to select the Platform.sh project to download, and then setup basic Lando configuration.
+You need hardware that meets the [requirements](https://docs.lando.dev/getting-started/installation.html#hardware-requirements).
 
-If you already have your project checked out locally, change into the project directory and run:
+Lando doesn't automatically pull and set up environment variables that have been set in the Console.
+To use a build hook that requires environment variables, manually [add them](https://docs.lando.dev/platformsh/config.html#environment-variables).
 
-```bash
-$ lando init --source cwd --recipe platformsh
-```
+## 1. Install Lando
 
-That will use your existing git clone and add the appropriate Lando configuration.
+Follow the [Lando installation instructions](https://docs.lando.dev/getting-started/installation.html).
 
-Either way, start the local environment by running:
+## 2. Initialize Lando
 
-```bash
-$ lando start
-```
+{{< codetabs >}}
 
-You can then download data from the environment for your current branch and load it into your local Lando environment by running:
+---
+title=On a new Platform.sh project without code
+file=none
+highlight=false
+---
+
+For a quicker start, create a project based on the Platform.sh [PHP template](https://github.com/platformsh-templates/php).
+The template provides the most basic configuration for running a custom PHP project built with Composer.
+It also includes the required Platform.sh configuration files out of the box.
+
+1. [Create a new project based on the PHP template](https://console.platform.sh/projects/create-project/template?query=php). <!-- TODO: The link probably has to change to have a specific source and campaign for everything `utm` -->
+2. Clone that project locally with Platform.sh [CLI](../../gettingstarted/introduction/own-code/cli-install.md) `platform get <PROJECT_ID>` or with [git](../../administration/web/_index.md#git).
+3. In the project's folder, run `lando init --recipe platformsh --source platformsh`.
+4. Follow the instructions provided by the interactive prompt.
+On the `Which project?` step, select the project created in the first step.
+
+<--->
+
+---
+title=On an existing Platform.sh project
+file=none
+highlight=false
+---
+
+If your code isn't present locally, retrieve your codebase with one of these methods:
+
+- Using the [Platform.sh CLI](../../gettingstarted/introduction/own-code/cli-install.md) by running `platform get <PROJECT_ID>`
+- Using [git](../../administration/web/_index.md#git)
```

Comment:
:hammer_and_wrench: Make the other way like this?

This description with a list is clearer than the one in the other tab. Maybe have that step have a sublist?

Reply:
Indeed, it's much clearer with a sublist! Good catch.